ctx_test.go

```diff
@@ -632,6 +632,23 @@ func Test_Ctx_FormValue(t *testing.T) {
 	utils.AssertEqual(t, StatusOK, resp.StatusCode, "Status code")
 }
 
+// go test -v -run=^$ -bench=Benchmark_Ctx_Fresh_StaleEtag -benchmem -count=4
+func Benchmark_Ctx_Fresh_StaleEtag(b *testing.B) {
```

Comment:
Benchmark shows no improvements or degradation. It's probably because the allocated string slice doesn't matter too much to the performance over all. But I think it's still worthing removing the unnecessary slice allocation and then iterate it.